README.md

```diff
@@ -159,3 +159,22 @@ else
   render :new
 end
 ```
+
+## Traps
+
+### session in your application is not yet loaded
+
+Sometimes, request for some simple pages in your Rails application may not create a session, as the session in Rails is created lazily. It will be created when it is explicitly used, like:
+
+```erb
+<%= csrf_meta_tags %>
+```
+or
+```rb
+# controller layer
+session[:some_key] = some_value
+```
+
+If not, you can get `#<ActionDispatch::Request::Session:0x7f9b28a670c8 not yet loaded>` when called `session` in controller. And the session key you seted will not exist in broswer.
+
+Rucaptcha use the session to distinguish the user or current captcha. So you need ensure the session is exist, when use Rucaptcha. Otherwise, it will let you in trouble.
```

Comment:
Keep this notice in this PR, I don't want too much text in README, that will let it looks like complex.